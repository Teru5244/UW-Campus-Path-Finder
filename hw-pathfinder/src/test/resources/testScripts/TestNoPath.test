# Test if Dijkstra's Algorithm works on two
# nodes that does not have an edge.

CreateGraph noPath

AddNode noPath A
AddNode noPath B
AddNode noPath C

AddEdge noPath A B 130.0
AddEdge noPath B A 130.0

FindPath noPath A C