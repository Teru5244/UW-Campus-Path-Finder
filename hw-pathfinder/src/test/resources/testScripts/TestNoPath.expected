# Test if Dijkstra's Algorithm works on two
# nodes that does not have an edge.

created graph noPath

added node A to noPath
added node B to noPath
added node C to noPath

added edge 130.000 from A to B in noPath
added edge 130.000 from B to A in noPath

path from A to C:
no path found